redbot/cogs/trivia/data/lists/worldflags.yaml

```
@@ -177,7 +178,7 @@ What country is represented by this flag? https://i.imgur.com/RTVaeeT.png:
 What country is represented by this flag? https://i.imgur.com/RgnmHkX.png:
 - Chile
 What country is represented by this flag? https://i.imgur.com/RhbzmE9.png:
-- Guinea-Bissau
+- Guinea Bissau
```

```suggestion
- Guinea-Bissau
- Guinea Bissau
```

Guinea-Bissau is recognized with a hyphen as it always has been, the hyphonated version needs to stay. The non-hyphonated version may also be added.